modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -1206,16 +1203,17 @@ protected function render() {
 		<div <?php $this->print_render_attribute_string( 'elementor-tabs' ); ?>>
 			<div class="e-n-tabs-heading" role="tablist">
 				<?php echo $tab_titles_html;// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
-			</div>
-			<div class="e-n-tabs-content">
+            </div>
```

Comment:
Tabs converted to spaces - hunting to see if this is an Storm bug or caused on CI. Converting spaces back to tabs.